sdk/core/Azure.Core/api/Azure.Core.net6.0.cs

```diff
@@ -17,7 +17,9 @@ public abstract partial class AsyncPageable<T> : System.Collections.Generic.IAsy
     }
     public static partial class AzureCoreExtensions
     {
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("This class utilizes reflection-based JSON serialization and deserialization which is not compatible with trimming.")]
```

Comment:
Does this also need `RequiresDynamicCode` as well?

Reply:
Hmm I have both on the method, not sure why it's not showing up in the API.

Reply:
Not sure why but the API doesn't include RequiresDynamicCode in the API for the existing attributes either - https://github.com/Azure/azure-sdk-for-net/blob/0e956ca58dc7bfd00431fdb3848aae3e4a3c0f76/sdk/core/Azure.Core/api/Azure.Core.net6.0.cs I'll have to follow up on it

Reply:
Ok, thanks for looking into it.